src/script/standard.cpp

```diff
@@ -57,10 +57,18 @@ static bool MatchPayToPubkeyHash(const CScript& script, valtype& pubkeyhash)
     return false;
 }
 
-/** Test for "small positive integer" script opcodes - OP_1 through OP_16. */
-static constexpr bool IsSmallInteger(opcodetype opcode)
+/** Test for "small positive integer" script opcodes - OP_1 through OP_16, or a single byte push for 17 to 20. */
```

Comment:
The old definition is used elsewhere still. Perhaps rename the function to "IsCheckMultisigInteger" or something more direct?

Reply:
I think it's only used here, and is newly introduced in  #13194

Reply:
I meant the term "small integer" i.e. "encode small integer" etc